Models/Effects/damage/left_gear.xml

```diff
@@ -28,8 +28,15 @@
 
 <PropertyList>
 
+  <params>
+    <!-- IMPORTANT: int properties 14 to 19, both included, are reserved for the immat system. Do not use them! -->
```

Comment:
It's not really necessary to put this warning here in all these effects files as long as it's put in `c172p-set.xml`. That's where you decide which multiplay property to use.